starter-code/index.html

```diff
@@ -9,6 +10,82 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
+
 <body>
+  <div class="container">
+    <h1 class='headline'>The Ironhack Cart</h1>
+    <div id="products">
+      <div class="product-line" id='hidden'>
+        <div class="product-name">
+          <span>Ironhack Skateboard</span>
+        </div>
+        <div class="product-cost">
+          <span class='unit-price'>50</span>
+          <p>$</p>
+        </div>
+        <div class="qty">
+          <label for="qty">QTY</label>
+          <input class='input' type="number">
```

Comment:
Little tip, if you don't want to allow negative values, do:
```
<input class="number" type="number" min="0">
```
Keep in mind this is not a perfect solution.